lib/compat/wordpress-6.1/get-global-styles-and-settings.php

```diff
@@ -75,15 +76,15 @@ function gutenberg_get_global_stylesheet( $types = array() ) {
 		( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) &&
 		! is_admin()
 	);
-	$transient_name = 'gutenberg_global_styles_' . get_stylesheet();
+	$transient_name = 'gutenberg_global_styles_' . $stylesheet;
 	if ( $can_use_cached ) {
 		$cached = get_transient( $transient_name );
 		if ( $cached ) {
 			return $cached;
 		}
 	}
 	$tree                = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data();
-	$supports_theme_json = wp_theme_has_theme_json();
+	$supports_theme_json = wp_theme_has_theme_json( $stylesheet );
```

Comment:
Do we need to do this? Isn't this code querying the active theme?

Reply:
https://github.com/WordPress/gutenberg/pull/45955/files#r1029270566